cost/google/rightsize_vm_recommendations/README.md

```diff
@@ -0,0 +1,54 @@
+# Google Rightsize VM Recommender
```

Comment:
New policies should be added to the Policy Template list in the top-level readme - https://github.com/flexera-public/policy_templates/blob/master/README.md